### DISCUSSION TITLE
Bottom bar redesign.

### DISCUSSION THREAD

**Discussion by simanchal-45** [robertodoering/harpy#709] [Show and tell]
Can we switch to this if possible?
![eee61ca3dd7d875a22417382655982b5](https://user-images.githubusercontent.com/71957533/177031565-ba3356e3-64c1-4c09-8ea1-c3c2e6f9d520.gif)

**Comment by HikaruIchijyo** [comment]
Nice design btw

**Comment by ghost** [comment]
Where is the design from?
looks pretty good, but maybe bit too much animations
